src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -27,7 +28,8 @@
 
     /** Default file path used if the user doesn't provide the file name. */
     public static final String DEFAULT_STORAGE_FILEPATH = "addressbook.xml";
-
+	public static final String MESSAGE_STORAGE_FILE_NOT_FOUND = "Storage File could not be found, it might have been deleted";
```

Comment:
Correct, you need new error message.

Note the indentation though, remember to set your "tab" to be "4 spaces" in IDE.